ngxunprivinst.sh

```diff
@@ -367,7 +382,13 @@ upgrade() {
         [ -d $TMPDIR/usr/lib64/ ] && cp -a $TMPDIR/usr/lib64/* $ABSPATH/usr/lib64/
         check_modules_deps
         TARGETVER=$($ABSPATH/usr/sbin/nginx -v 2>&1 | cut -d '(' -f 2 | cut -d ')' -f 1 | cut -d'-' -f 3 | tr -d 'r')
-        if [ $TARGETVER -ge 31 ]; then
+        if [ $TARGETVER -ge 33 ]; then
+            if ! $ABSPATH/usr/sbin/nginx -p $ABSPATH/etc/nginx -c nginx.conf -T 2>&1 | grep 'license_token' | grep -vE '^(.*)#.*license_token' >/dev/null; then
```

Comment:
As I understand it, with this kind of check, I’ll end up with a broken conf if I already have `mgmt{}` block, say, from a previous ver? And will get somthing like:
```
mgmt {uuid_file foo;}
mgmt {license_token bar;}
```

Reply:
btw, in the case of upgrade, we still need to have `report_state_file`. However, if we’re upgrading from 32->33, it creates a deadlock situation -- we’ have to remove `uuid_file` from cfg and add `report_state_file` along with `license_token`. But in this case, the unique id will be lost, and the nginx instance will be considered new…

Reply:
Hmm, yes, upgrade from R31/R32 to R33 will be broken if the configuration already has an mgmt block.  I'll fix it.

Reply:
I can’t commit directly to this PR, so I sent you these changes for review in your fork.